GUI/Main/Main.cs

```diff
@@ -765,17 +767,45 @@ public void LaunchGame()
             var binary = split[0];
             var args = string.Join(" ", split.Skip(1));
 
+            gameTime = new Stopwatch();
+
+            TimeLog t = new TimeLog(CurrentInstance.CkanDir());
             try
             {
+                
                 Directory.SetCurrentDirectory(CurrentInstance.GameDir());
-                Process.Start(binary, args);
+
+                Process kspProcess = new Process();
+                kspProcess.StartInfo.FileName = binary;
+                kspProcess.Exited += new EventHandler(KspExit);
+                kspProcess.EnableRaisingEvents = true;
+
+                kspProcess.Start();
+                gameTime.Start();
             }
             catch (Exception exception)
             {
                 currentUser.RaiseError(Properties.Resources.MainLaunchFailed, exception.Message);
             }
         }
 
+        private void KspExit(object sender, System.EventArgs e)
+        {
+            gameTime.Stop();
+            TimeSpan ts = gameTime.Elapsed;
+
+            TimeLog t = new TimeLog(CurrentInstance.CkanDir());
+            t.InstanceName = CurrentInstance.Name;
+            t.addTime((int)ts.TotalSeconds);
+            
+            t.Save();
+
+            Console.WriteLine(t.Time);
+            Console.WriteLine(t.InstanceName);
```

Comment:
Don't `Console.WriteLine`. If you need to print something, use one of the `log4net` objects.

Reply:
I only had that there for testing for got to remove it